src/coreclr/ilasm/main.cpp

```diff
@@ -16,15 +16,40 @@
 
```

Comment:
Side note: if we are not going to rewrite ilasm/ildasm in C# in a near future, it would be nice to combine ilasm/main.cpp and ildasm/windasm.cpp main() functions and the arguments parsing (~1.5K LOC).

Reply:
That is a solid idea. It might be worth creating an issue for that.